vm-image-spec.yaml

```diff
@@ -142,6 +142,32 @@ files:
         query: |
           select datname, state, count(*) as count from pg_stat_activity where state <> '' group by datname, state;
 
+      - metric_name: pg_stats_userdb
+        type: gauge
+        help: 'Stats for the oldest non-system db'
+        key_labels:
+          - datname
+        value_label: kind
+        values:
+          - db_size
+          - deadlocks
+          # Rows
+          - inserted
+          - updated
+          - deleted
+        query: |
+          select pg_database_size(datname) as db_size, deadlocks,
+                 tup_inserted as inserted, tup_updated as updated, tup_deleted as deleted,
+                 datname
+            from pg_stat_database
+           where datname IN (
+             select datname
+               from pg_database
+              where datname <> 'postgres' and not datistemplate
+              order by oid
+              limit 1
```

Comment:
what will happen if we remove this limit?

Average number of databases per branch

```
neon_console=> select avg(cnt) from (select count(*) cnt from databases group by branch_id) a;
        avg
--------------------
 1.1252440084040954
(1 row)

Time: 798.818 ms
```

but there are some extremums 

```
neon_console=> select count(*) cnt from databases group by branch_id order by cnt desc limit 10;
 cnt
-----
 925
 421
 313
 204
 168
 160
 132
 110
 105
  99
(10 rows)
```

Reply:
I think for the MVP we need this limit. Without this limit it is too easy to abuse the system by creating a lot of databases.
I am all in to lifting this limit in the future after we understand the needs better.

Reply:
may I ask you to leave this comment right in the yaml file?